sdk/identity/identity/package.json

```diff
@@ -36,7 +36,7 @@
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint package.json src test --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint package.json src test --ext .ts -f html -o identity-lintReport.html",
+    "lint": "eslint package.json src test --ext .ts",
```

Comment:
Are we perhaps losing out on the benefits of the html report? Related issue: #4344

Reply:
I guess the html helps when there are many errors and we want to go through them one by one. Assuming that we are now lint error free, future errors are unlikely to be large enough in number to require an html view